quesma/model/bucket_aggregations/geotile_grid.go

```diff
@@ -44,6 +44,18 @@ func (query GeoTileGrid) TranslateSqlResponseToJson(rows []model.QueryResultRow,
 	}
 }
 
+func (query GeoTileGrid) extractFloat(value any) float64 {
+	switch valueTyped := value.(type) {
+	case float64:
+		return valueTyped
+	case float32:
+		return float64(valueTyped)
+	default:
+		logger.ErrorWithCtx(query.ctx).Msgf("unexpected value type in geotile_grid aggregation response: %T", value)
+		return 0
+	}
+}
+
```

Comment:
If a column is of `Nullable(Float)` type, it'll return `*float`. Will it always be just `Float` here?
Maybe use this function from `util/utils.go`?
```
func ExtractFloat64(value any) float64 {
	switch valueTyped := value.(type) {
	case float64:
		return valueTyped
	case *float64:
		return *valueTyped
	case float32:
		return float64(valueTyped)
	case *float32:
		return float64(*valueTyped)
	}
	logger.Error().Msgf("ExtractFloat64, value of incorrect type. Expected (*)float64, received: %v; type: %T", value, value)
	return -1
}
```

Reply:
Fixed.